release notes/v0.9.0.md

```diff
@@ -0,0 +1,111 @@
+k6-browser `v0.9.0` is here 🎉! This release includes:
+
+- A few breaking changes.
+- `browserContext.addCookies` has now been implemented.
+- `browserType.Connect` has now been implemented and so k6-browser can connect to an already running Chrome/Chromium browser instance.
```

Comment:
Providing a suggestion just in case this sentence was mistyped for some reason, but of course I trust you way more than me on writing in English :smile: 
```suggestion
- `browserType.Connect` has been implemented so k6-browser can now connect to an already running Chrome/Chromium browser instance.
```